app/shared/src/badge/AdaptableBadge.js

```diff
@@ -50,7 +62,11 @@ export default function AdaptableBadge(props: Props) {
     <View style={[style.container, props.style]}>
       {props.icon}
       <Text style={[style.text, props.textStyle]}>
-        <Translation passThrough={props.text} />
+        {props.text !== undefined ? (
+          <Translation passThrough={props.text} />
+        ) : (
+          props.translation
```

Comment:
Can we just be strict here and accept only translations instead of this? (maybe send separate PR first)

Reply:
Yes, I will create separate PR for this 👍